railties/lib/rails/generators/app_base.rb

```diff
@@ -340,7 +340,15 @@ def spring_install?
       end
 
       def run_bundle
-        bundle_command('install') if bundle_install?
+        # Most of us have the CPUs; so let's parallelize `bundle install`
```

Comment:
I am not an OS guy, but as far as I know cores map to threads, not to processes.

That is, a computer runs as many *threads* simultaneously as cores (may be logical cores), the rest are waiting to be scheduled. Every process starts with one thread, that is the one that runs. From the OS point of view the total number of threads going on is the same if you have N single-threaded processes or 1 process with N threads, and the CPU. So I would expect the number of cores to be relevant in either case.

Processes and threads have different characteristics, in particular RAM usage, but that is a different matter.

Am I wrong?